packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -312,11 +324,18 @@ function Preview({
     setIsMultiTouching(false);
   }
 
+  function onWrapperMouseWheel(e: WheelEvent<HTMLDivElement>) {
+    e.preventDefault();
+    setIsPressing(false);
+    setIsMultiTouching(false);
```

Comment:
Just curious: why do we want to cancel the touch when a scroll occurs?

Reply:
Copied from the function above, forgot to verify this before committing. At first I thought it would help prevent zooming when multitouch is active, but just tested that it only hides the multitouch circles, so removed this in e02c7f0. As for pressing, it seems not to interfere with scrolling, so I removed this `set` as well.